README.md

```diff
@@ -1,5 +1,25 @@
-# HuJSON - Human JSON
+# HuJSON - "Human JSON" ([JWCC](https://nigeltao.github.io/blog/2021/json-with-commas-comments.html))
+
+The `github.com/tailscale/hujson` package implements
+the [JWCC](https://nigeltao.github.io/blog/2021/json-with-commas-comments.html) extension
+of standard JSON ([RFC 8259](https://datatracker.ietf.org/doc/html/rfc8259)).
+
+The `JWCC` format permits two things over standard JSON:
+
+1. C-style line comments and block comments intermixed with whitespace,
+2. allows trailing commas after the last member/element in an object/array.
+
+All JSON is valid JWCC.
+
+For details, see the JWCC docs at:
+
+    https://nigeltao.github.io/blog/2021/json-with-commas-comments.html
+
+## History
+
+JWCC and HuJSON were written at the same time, unaware of each
```

Comment:
I don't think this is true. HuJSON predates JWCC and I believe Nigel was aware of it. He picked another name because he didn't like JWCC. (Though @nigeltao can correct me if I'm wrong.)

Reply:
The JWCC page says:

> In a case of parallel evolution, Tailscale already have a Go implementation of this format. They call it HuJSON - Human JSON.

So I was just writing the ~same text here, assuming that was true.

Reply:
Just removed that part for now.